web/src/containers/ActivityDetailCostAllocateFFP.js

```diff
@@ -38,7 +38,7 @@ class ActivityDetailCostAllocateFFP extends Component {
           {byYearData.map(
             ({ year, total, totalNetOther, ffpSelectVal, allocations }) => (
               <div key={year} className="col col-12 sm-col-4 px1">
-                <div className="p2 bg-darken-1">
+                <div className="p2 bg-gray-lightest">
```

Comment:
Colors with alpha seem to cause problems for HTML_CodeSniffer, so I switched over to an absolute color.  It was complaining that _nearly white_ was too low contrast with _actually black_.  🤷‍♂️